Upstream: No
Reason: Skip test that can't be run with -buildmode=pie
--- a/integration/flags_test.go
+++ b/integration/flags_test.go
@@ -106,6 +106,7 @@ var _ = Describe("Flags Specs", func() {
 	})
 
 	It("should run the race detector when told to", func() {
+		Skip("-buildmode=pie not supported when -race is enabled")
 		if !raceDetectorSupported() {
 			Skip("race detection is not supported")
 		}
